client/lib/Hero.ts

```diff
@@ -465,8 +452,7 @@ class SessionConnection {
   constructor(private hero: Hero, stateOptions: IStateOptions) {
     const { connectionToCore, ...options } = stateOptions;
 
-    // @ts-ignore
-    const createConnectionToCoreFn: ICreateConnectionToCoreFn = hero.constructor.createConnectionToCore;
+    const createConnectionToCoreFn = (hero.constructor as any).createConnectionToCore;
```

Comment:
I don't understand this code. Is it just saying `Hero.createConnectionToCore`? You can't change it per instance right?

Reply:
Look at fullstack/lib/Hero.ts. It extends the Hero class and sets ConnectionToLocalCore on createConnectionToCore. Therefore, I can't refer to Hero.createConnectionToCore. I must respect the child class.

Reply:
Oh, I see. Should we have a definition on the base class that just doesn't have an implementation? Kind of seems like an abstract variable of sorts.

Reply:
I added to IHero.

Reply:
but yes, I will add.

Reply:
Good idea, will add.

Reply:
Added